tests/compiler/std/array.ts

```diff
@@ -313,6 +313,19 @@ var i: i32;
   assert(([NaN] as f64[]).indexOf(NaN) == -1);
 }
 
+// Array#lastIndexOf
+
+{
+  const numbers: i32[] = [2, 5, 9, 2];
+  assert(numbers.lastIndexOf(2) == 3);
+  assert(numbers.lastIndexOf(7) == -1);
+  assert(numbers.lastIndexOf(2, 3) == 3);
+  assert(numbers.lastIndexOf(2, 2) == 0)
```

Comment:
```suggestion
  assert(numbers.lastIndexOf(2, 2) == 0);
```

Reply:
Yeah good catch, this was causing CI to fail.